cmd/update/cluster/command.go

```diff
@@ -0,0 +1,88 @@
+package cluster
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name = "cluster --name <cluster-name> --namespace <cluster-namespace> --release-version <release-version> --scheduled-time <scheduled-time> --provider <provider>"
```

Comment:
can we maybe use a better const name here? `name` sounds wrong to me.

Reply:
This is the same const name like any other subcommand description, would it be ok for you to align with the other ones? Otherwise we need to change it everywhere.

Here's what I mean some examples: 
- https://github.com/giantswarm/kubectl-gs/blob/master/cmd/update/app/command.go#L18
- https://github.com/giantswarm/kubectl-gs/blob/master/cmd/login/command.go#L18

Reply:
all good